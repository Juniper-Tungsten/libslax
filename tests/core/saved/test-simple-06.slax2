version 1.1;

match doc {
    var $location = location;
    
    <top> {
        <element attr1="one" attr2="two">;
        <location state=$location/state zip=$location/zip5 _ "-" _ $location/zip4>;
        <avt sign="{here}">;
    }
}
