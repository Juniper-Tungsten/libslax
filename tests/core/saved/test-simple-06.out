<?xml version="1.0"?>
<top>
  <element attr1="one" attr2="two"/>
  <location state="nc" zip="27614-1312"/>
  <avt sign="{here}"/>
</top>
